Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -192,6 +194,9 @@ - (void)startListeningToNotifications {
 
     // Themes
     [nc addObserver:self selector:@selector(themeDidChange) name:SPSimplenoteThemeChangedNotification object:nil];
+
+    // App Background
+    [nc addObserver:self selector:@selector(appWillEnterBackground) name:UIApplicationDidEnterBackgroundNotification object:nil];
```

Comment:
Nit: `UIApplicationWillEnterForegroundNotification` and you can remove the AppDelegate from the equation!

Reply:
yeah, good call.  Done